polkadot-protocol/architecture/system-chains/coretime.md

```diff
@@ -1,4 +1,62 @@
+
+The specification for the Coretime system chain, along with Coretime as a concept, can be found in the Fellowship RFC: [RFC-1: Agile Coretime](https://github.com/polkadot-fellows/RFCs/blob/main/text/0001-agile-coretime.md){target=_blank}. 
+
+Besides core management, its responsibilities include: 
+
+- The number of cores that should be made available
+- Which tasks should be running on which cores and in what ratios
+- Accounting information for the on-demand pool
+
+From the relay chain, it expects the following via Downward Message Passing (DMP):
+
+- The number of cores available to be scheduled
+- Account information on on-demand scheduling
+
+The details for this interface can be found in [RFC-5: Coretime Interface](https://github.com/polkadot-fellows/RFCs/blob/main/text/0005-coretime-interface.md){target=_blank}.
+
+## Bulk Coretime Assignment
+
+The Coretime chain allocates coretime before its usage. It also manages the ownership of a core. As cores are made up of regions (by default, one core is a single region), a region is recognized as a non-fungible asset. The Coretime chain exposes Regions over XCM as an NFT. Users can transfer individual regions, partition, interlace, or allocate them to a task. Regions describe how a task may use a core.
+
+!!!info "One core can contain more than one region."
+ A core can be considered a logical representation of an active validator set on the relay chain, where these validators commit to verifying the state changes for a particular task running on that region. With partitioning, having more than one region per core is possible, allowing for different computational schemes. Therefore, running more than one task on a single core is possible.
+
+<!-- TODO: Some sort of diagram of this would be pretty helpful, maybe -->
+
+### Assigning Region
+
+Regions can be assigned to a task on the relay chain, such as a parachain/rollup. 
+
+### Transferring Regions
+
+Regions may be transferred on the Coretime chain, upon which the [`transfer`](https://paritytech.github.io/polkadot-sdk/master/pallet_broker/pallet/dispatchables/fn.transfer.html){target=_blank} [dispatchable](../../glossary.md#dispatchable) in the Broker pallet would assign a new owner to that specific region.
+
+### Partitioning Regions
+
+Regions may be partitioned into two non-overlapping subregions within the same core. A partition involves specifying a *pivot*, wherein the new region will be defined and available for use.
+
+### Interlacing Regions
+
+Interlacing a region adds the ability to have alternative-compute strategies for a core. Whereas partitioned regions are non-overlappable, interlaced regions overlap because multiple tasks may utilize a single core alternatingly.
```

Comment:
Having these headers for such short information seems strange from a visual perspective. Maybe you can add more details on each section to make them longer or just replace this structure with list items. What do you think?

Reply:
List items sounds good to me, will go with that